### PR TITLE
add github star and  fork icons in home page 

### PR DESCRIPTION
**Pull request by lovepoem**
## What is the purpose of the change

add github star and  fork icons in home page 
https://github.com/apache/incubator-dubbo-website/issues/69

## Brief changelog

	modified:   src/pages/home/index.jsx
	modified:   src/pages/home/index.scss
image

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/incubator-dubbo-website/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Format the pull request title like `Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Test your code locally by running `docsite start`, and make sure it works as expected.
- [ ] Make sure no files under build directory is added.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/incubator-dubbo/wiki/Software-donation-guide).

### CODE REVIEW

**Comment by ralf0131** [comment]
Looking at the screenshot, the icons do not align with the above two buttons `Get Started` and `GITHUB`. Shall we align with it?

**Comment by lovepoem** [comment]
image
Updated

File: src/pages/home/index.jsx [hunk 1]
```diff
@@ -61,6 +73,24 @@ class Home extends Language {
               <Button type="primary" link={getLink(dataSource.brand.getStartedButton.link)}>{dataSource.brand.getStartedButton.text}</Button>
               <Button type="normal" link={getLink(dataSource.brand.viewOnGithubButton.link)}>{dataSource.brand.viewOnGithubButton.text}</Button>
             </div>
+            <div className="github-buttons">
+              <a href="https://github.com/apache/incubator-dubbo">
+                <div className="star">
+                  <img src="https://img.alicdn.com/tfs/TB1FlB1JwHqK1RjSZFPXXcwapXa-32-32.png" />
```
Comment (ralf0131) [inline src/pages/home/index.jsx]: Would you please inline this png file? Putting on a cdn may be unfriendly to foreign visitors.

File: src/pages/home/index.jsx [hunk 2]
```diff
@@ -61,6 +73,24 @@ class Home extends Language {
...
+                  <span className="type">Star</span>
+                  <span className="line" />
+                  <span className="count">{starCount}</span>
+                </div>
+              </a>
+              <a href="https://github.com/apache/incubator-dubbo/fork">
+                <div className="fork">
+                  <img src="https://img.alicdn.com/tfs/TB1zbxSJwDqK1RjSZSyXXaxEVXa-32-32.png" />
```
Comment (ralf0131) [inline src/pages/home/index.jsx]: And this file.

**Comment by caojiele** [comment]
@lovepoem Button layout in the wrong.
image

**Comment by caojiele** [comment]
> @lovepoem Button layout in the wrong.
> image

And I think there is no need to have a Star and Fork dead simple.For example:
image

**Comment by lovepoem** [comment]
I think it's because of the cache .
Please clean the cache of web browser and try again

Jlcao <notifications@github.com> 于2019年3月5日周二 下午12:08写道：

**Comment by ralf0131** [comment]
Hi, the font looks different with others, are they built-in styles?

**Pull request merged**